### ISSUE TITLE
Provide textual confirmation of validity

### ISSUE THREAD

**Issue by greywillfade** [openactive/model-validator#8]
The current tool confirms validity through the use of a green border around the textarea, whereas invalid JSON triggers a "You have X errors" message. 

For accessibility reasons colour or UI changes alone should not be used to convey meaning, and some kind of textual message should be included.
